.github/workflows/generate-instance.yml

```diff
@@ -20,10 +20,10 @@ jobs:
           persist-credentials: false # Required to make github pages deployment work correctly
 
       - name: Cache Data # Cache SourceCred Data, invalidating if any of the config changes or the SC version is updated
-        uses: actions/cache@v2
+        uses: pat-s/always-upload-cache@v2.1.5
```

Comment:
can you remove this from this pr since our priority is currently to fix the recalc and this is more likely to break it than fix it?
otherwise lgtm

Reply:
No, this is what will allow it to actually pick up where it left off. The PR is worthless without this

Reply:
IMO the cache change is entirely unrelated to the calc issue. But we cannot do the calc at all without the cache persisting after a failed action (i.e. 6hr timeout)

Reply:
> IMO the cache change is entirely unrelated to the calc issue. But we cannot do the calc at all without the cache persisting after a failed action (i.e. 6hr timeout)

Let's merge and hopefully get this issue resolved.